### PR TITLE
install vision in devel dockerfile, minor fixes to dockerfile

### LINKED ISSUE #2009: Docker image does not contain torchvision
Torchvision is considered a basic requirement of the tutorials. Perhaps it makes sense to include it in the docker build.

```
# python
Python 3.5.2 |Continuum Analytics, Inc.| (default, Jul  2 2016, 17:53:06)
[GCC 4.4.7 20120313 (Red Hat 4.4.7-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import torchvision
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named 'torchvision'

```

### PR DESCRIPTION
**Pull request by ngimel**
Fix #2009

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -32,5 +30,7 @@ RUN TORCH_CUDA_ARCH_LIST="3.5 5.2 6.0 6.1+PTX" TORCH_NVCC_FLAGS="-Xfatbin -compr
     CMAKE_PREFIX_PATH="$(dirname $(which conda))/../" \
     pip install -v .
 
+RUN git clone https://github.com/pytorch/vision.git && cd vision && pip install -v .
```
Comment (alykhantejani) [inline Dockerfile]: Do we want this to be torchvision master, or should it be the latest stable release i.e. `conda install torchvision -c soumith` as this is a pytorch-dev dockerfile?

Comment (soumith) [inline Dockerfile]: the dockerfile installs the `master` pytorch, and probably deserves the `master` torchvision.

**Review by soumith — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (ngimel) [inline Dockerfile]: In devel dockerfile, pytorch is compiled from master, so it makes sense to have the latest torchvision too. Runtime dockerfile installs latest conda packages.

Comment (alykhantejani) [inline Dockerfile]: Cool - makes sense

**Pull request merged**